src/plugins/unified_histogram/public/chart/chart.tsx

```diff
@@ -136,7 +136,7 @@ export function Chart({
     !chart.hidden &&
     dataView.id &&
     dataView.type !== DataViewType.ROLLUP &&
-    dataView.isTimeBased()
+    (isPlainRecord || (!isPlainRecord && dataView.isTimeBased()))
```

Comment:
ℹ️ This was a bug, the chart in text based languages should render regardless if it is timeBased or not as we don't show the histogram but the Lens suggestions